discord/http.py

```diff
@@ -1476,13 +1476,7 @@ def get_guild(
     def delete_guild(self, guild_id: Snowflake) -> Response[None]:
         return self.request(Route("DELETE", "/guilds/{guild_id}", guild_id=guild_id))
 
-    def create_guild(self, name: str, icon: str | None) -> Response[guild.Guild]:
```

Comment:
The icon parameter is removed, was it used anywhere in the library ? Just make sure this is changed everywhere it should be. Otherwise this is fine and not considered breaking.

Reply:
The icon parameter is a key for ``payload`` so it is not required as a parameter anymore, and no, this method is only used on ``Client.create_guild``.